admin_bot/view_schedule/handlers.py

```diff
@@ -78,8 +97,49 @@ def show_trainingroupday_info(update, context: CallbackContext):
     players_info = f"{group_name}{group_level}{group_players}{visitors}{pay_visitors}{pay_bonus_visitors}{absents}"
     text = f"{general_info}{players_info}"
 
-    markup = back_from_show_grouptrainingday_info_keyboard(
-        year=tr_day.date.year, month=tr_day.date.month, day=tr_day.date.day
-    )
+    markup = show_grouptrainingday_available_change_keyboard(tr_day)
+
+    bot_edit_message(context.bot, text, update, markup)
+
+
+def confirm_change_available_status_handler(update: Update, context: CallbackContext):
+    """
+    Обработчик для создания подтверждения перед изменением: available_status у дня по его id
+    """
+    query = update.callback_query
+    (purpose, action, id) = separate_callback_data(query.data)
+    markup = show_grouptrainingday_available_change_confirm_keyboard(action, id)
```

Comment:
```suggestion
    _, action, tr_day_id = separate_callback_data(query.data)
    markup = show_grouptrainingday_available_change_confirm_keyboard(action, tr_day_id)
```
1. если переменная не используется, заменяй ее на "_"
2. id -- системное название функции из названия непонятно, чей именно id